tests/conftest.py

```diff
@@ -2,22 +2,109 @@
 import importlib
 import pytest
 import requests
+from sqlalchemy import create_engine
 import mock
 from unittest.mock import patch
 
 # indexd_server and indexd_client is needed as fixtures
-from cdisutilstest.code.indexd_fixture import clear_database
 from gen3authz.client.arborist.client import ArboristClient
 
 from indexd import get_app
 from indexd import auth
 from indexd.auth.errors import AuthError
-from tests import default_test_settings
+from indexd.index.drivers.alchemy import Base as index_base
+from indexd.auth.drivers.alchemy import Base as auth_base
+from indexd.alias.drivers.alchemy import Base as alias_base
+from indexd.index.drivers.alchemy import SQLAlchemyIndexDriver
+from indexd.alias.drivers.alchemy import SQLAlchemyAliasDriver
+from indexd.auth.drivers.alchemy import SQLAlchemyAuthDriver
+
+POSTGRES_CONNECTION = "postgresql://postgres:postgres@localhost:5432/indexd_tests"  # pragma: allowlist secret
+
+
+def clear_database():
+    """
+    Clean up test data from unit test
+    """
+    engine = create_engine(POSTGRES_CONNECTION)
+
+    with engine.connect() as conn:
+        index_driver = SQLAlchemyIndexDriver(POSTGRES_CONNECTION)
+        # IndexD table needs to be delete in this order to avoid foreign key constraint error
+        table_delete_order = [
+            "index_record_url_metadata",
+            "index_record_url",
+            "index_record_hash",
+            "index_record_authz",
+            "index_record_ace",
+            "index_record_alias",
+            "index_record_metadata",
+            "alias_record_hash",
+            "alias_record_host_authority",
+            "alias_record",
+            "index_record",
+            "drs_bundle_record",
+            "base_version",
+            "record",
+        ]
+
+        for table_name in table_delete_order:
+            delete_statement = f"DELETE FROM {table_name}"
+            conn.execute(delete_statement)
+
+        # Clear the Alias records
+        alias_driver = SQLAlchemyAliasDriver(POSTGRES_CONNECTION)
+        for model in alias_base.__subclasses__():
+            table = model.__table__
+            delete_statement = table.delete()
+            conn.execute(delete_statement)
+
+        # Clear the Auth records
+        auth_driver = SQLAlchemyAuthDriver(POSTGRES_CONNECTION)
+        for model in auth_base.__subclasses__():
+            table = model.__table__
+            delete_statement = table.delete()
+            conn.execute(delete_statement)
+
+
+@pytest.fixture(scope="function", params=["default_settings", "single_table_settings"])
+def combined_default_and_single_table_settings(request):
```

Comment:
There are tests that shouldn't use the fixture

Reply:
for tests that shouldn't use the fixture, shouldn't we use the single table by default instead of the multitable? [here](https://github.com/uc-cdis/indexd/blob/bad7c84aa036d2471bb9be99a5c049521be595ab/tests/conftest.py#L110-L115)
since single table will eventually be the default